precode.go

```diff
@@ -39,14 +40,67 @@ var tasks = map[string]Task{
 	},
 }
 
-// Ниже напишите обработчики для каждого эндпоинта
-// ...
+// Обработчик для получения всех задач
+func getAllTasks(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	tasksSlice := make([]Task, 0, len(tasks))
+	for _, task := range tasks {
+		tasksSlice = append(tasksSlice, task)
+	}
+	if err := json.NewEncoder(w).Encode(tasksSlice); err != nil {
```

Comment:
В промежуточном слайсе смысла нет, можно просто серилизовать изначальный массив
```suggestion
	if err := json.NewEncoder(w).Encode(tasks); err != nil {
```